### PR TITLE
Show how many participants there are #42

### PR DESCRIPTION
**Pull request by oanalavinia**
Co-Authored-By: Gabriel Raileanu <raileanu.gabriel97@gmail.com>

### CODE REVIEW

File: application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml [hunk 1]
```diff
@@ -142,7 +144,7 @@
           ## Table header row.
           &lt;thead&gt;
             &lt;tr&gt;
-              &lt;th&gt; $services.localization.render('contrib.xpoll.user') &lt;/th&gt;
+              &lt;th&gt; $services.localization.render('contrib.xpoll.user', $usersCount) &lt;/th&gt;
```
Comment (vmassol) [inline application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml]: I'm curious: why include the total as part of the translation and not something that is displayed after for example?

For example:

```
&lt;th&gt; $services.localization.render('contrib.xpoll.user') ($usersCount) &lt;/th&gt;
```

Or styled differently with CSS

?

In this way you wouldn't:
* break the existing translation key
* allow it to work for any translation
* the count doesn't really seem to be a translation, what is there to translate?

Thx

Comment (vmassol) [reply application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml]: Re the CSS styling I was thinking about displaying the count in a different style, after the text. You wouldn't be able to do that in a translation string.

Comment (oanalavinia) [reply application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml]: Indeed, there is no need of breaking the translation key if what is introduced it's a number. Thanks!

Regarding de css, for me the number is easy to be seen. What are you thinking for highlighting it? 
@evalica WDYT?

**Comment by vmassol** [comment]
> Regarding the css, for me the number is easy to be seen. What are you thinking for highlighting it?

The idea would be to separate visually the text from the number. Maybe some sort of bubble around the number with a different background color.

**Comment by oanalavinia** [comment]
After talking with @evalica I made changes just for using a grey color, without replacing the parentheses
votess
since this is a consistent style we are using in XWiki for showing a count (i.e. comments, attachments count).

**Comment by vmassol** [comment]
I still think it's not the right approach to have the number in the translation. Do you agree?

**Comment by vmassol** [comment]
I listed several reasons (breakage of translation, the fact that it won't work for translations that are not updated) and again the fact that if you want to style the number differently or position it in a different location, you won't be able to do that.

**Comment by oanalavinia** [comment]
Yes, to have the number in the translation wasn't a good idea and I have changed it to be separated in a new PR https://github.com/xwikisas/application-xpoll/pull/45

**Pull request merged**